aries_cloudagent/ledger/routes.py

```diff
@@ -590,20 +594,29 @@ async def ledger_accept_taa(request: web.BaseRequest):
             raise web.HTTPForbidden(reason=reason)
 
     accept_input = await request.json()
+    LOGGER.info(f">>> accepting TAA with: {accept_input}")
```

Comment:
I think this is often flagged as a linting error, the log methods are expected to format the arguments for you so it's cheaper when the output is skipped. ie. `LOGGER.info(">>> accepting TAA with: %s", accept_input)`

Reply:
got it thanks